service/auth/README.md

```diff
@@ -8,18 +8,71 @@ Users can easily add oauth2 providers to login.
 
 ## Start
 
-@see: depoy Kubefile
+### Step by step installation and run
```

Comment:
Please make sure that every code block has an associated language with it, unless there's no associated languages. For example:

<pre>
```shell
sealos run ...
```
</pre>